agency-api-spec.md

```diff
@@ -72,94 +104,136 @@ JSON payload that contains the form fields.
 
 #### Request fields
 
+**Field:** | `request_id`
+:--- |:---
+**Type:** | integer
+**Description:** | A unique identifier for the request within the Portal.
+**Required:** | yes
+**Example:** | `1543`
+
 **Field:** | `agency_name`
 :--- |:---
-**Type:** | text
+**Type:** | string
 **Description:** | Name of the tier 1 agency.
 **Required:** | yes
 **Example:** | `"Department of Justice"`
 
 **Field:** | `agency_component_name`
 :--- |:---
-**Type:** | text
+**Type:** | string
 **Description:** | Name of the department, bureau, or office.
 **Required:** | yes
 **Example:** | `"Office of Information Policy"`
 
-**Field:** | `requester_name`
+**Field:** | `name_first`
 :--- |:---
-**Type:** | object
-**Description:** | Full name of the requester.
-**Required:** | yes
-**Example:** | `{"first": "George", "last": "Washington"}`
+**Type:** | string
+**Description:** | First name of the requester.
+**Required:** | no
+**Example:** | `"George"`
 
-**Field:** | `requester_address`
+**Field:** | `name_last`
 :--- |:---
-**Type:** | object
-**Description:** | Mailing address of the requester.
-**Required:** | yes
-**Example:** | `{"address1": "1800 F Street", "address2": "Suite 400", "city": "Mount Vernon", "state": "Virginia", "zip": "98273"}`
+**Type:** | string
+**Description:** | Last name of the requester.
+**Required:** | no
+**Example:** | `"Washington"`
+
+**Field:** | `address_line1`
+:--- |:---
+**Type:** | string
+**Description:** | Requester’s street mailing address.
+**Required:** | no
+**Example:** | `"1800 F Street"`
 
-**Field:** | `description`
+**Field:** | `address_line2`
 :--- |:---
-**Type:** | text
+**Type:** | string
+**Description:** | Line 2 for requester’s mailing address.
```

Comment:
After talking things through with @RyanSibley, we were thinking the two address fields would be changed to "Mailing Address" and "Apartment or suite".

Although now that I've thought it through a bit more, aren't there scenarios where folks may want to use two address lines and not provide an 'apartment or suite'? In that case Mailing Address Line 1 and Mailing Address Line 2 may be the safe option.

@RyanSibley, @adborden what do you think?

Reply:
Checking on this @malikkotob I'll get back to you soon.

Reply:
I saw we leave it as is. This just a machine-readable name, it doesn't have to match the label that requesters will see. I think line1/line2 is fine for this.

Reply:
👍